service/src/test/java/uk/gov/hmcts/reform/fpl/service/HearingBookingServiceTest.java

```diff
@@ -132,29 +123,79 @@ private DynamicListElement createDynamicElement(UUID code) {
         }
     }
 
+    @Nested
+    class GetPastHearings {
```

Comment:
maybe edge case where hearing date is today should be tested

Reply:
hmm, what should be expected here. Now will always be in the past by the time the assertion runs 🤔